### DISCUSSION TITLE
404 Error on install - Kali Linux

### DISCUSSION THREAD

**Discussion by skilfoy** [apache/couchdb#3894] [Q&A] [answered]
Hello,

I am trying to install couchdb on Kali Linux. I'm following the directions on [The page for Installation on Unix-like systems](https://docs.couchdb.org/en/stable/install/unix.html).

All of the steps proceed without error until sudo apt update, when I receive the following error:
└─$ sudo apt update
Ign:2 https://apache.jfrog.io/artifactory/couchdb-deb kali-rolling InRelease
Err:3 https://apache.jfrog.io/artifactory/couchdb-deb kali-rolling Release
  404  Not Found [IP: 54.148.141.177 443]
Hit:1 http://kali.download/kali kali-rolling InRelease
Reading package lists... Done
E: The repository 'https://apache.jfrog.io/artifactory/couchdb-deb kali-rolling Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

I tried deleting the contents of /etc/apt/sources.list.d and the /usr/share/keyrings/couchdb-archive-keyring.gpg file and starting over, but I encounter the same error. Can someone please assist me with this error?

Thanks,
Sean

**Comment by big-r81** [comment]
Hi,

maybe it was a network or server error?

**Reply by skilfoy** [reply]
Nope, I can ping and navigate everywhere else. I think the repo for couchdb has been updated.

**Comment by big-r81** [comment, marked as answer]
Did you follow these guide?: https://docs.couchdb.org/en/latest/install/unix.html#installation-using-the-apache-couchdb-convenience-binary-packages
